From d2b260d19b2f1b877ad990d9e8bc32152c8e108a Mon Sep 17 00:00:00 2001
From: Ubuntu <ubuntu@ip-172-31-27-164.ap-northeast-2.compute.internal>
Date: Tue, 12 Feb 2019 13:56:43 +0900
Subject: [PATCH 3/3] Specified Android Nougat 7.0 (API 24)

This commit is to specifiy the api version of Android Nougat 7.0.

Signed-off-by: Ubuntu <ubuntu@ip-172-31-27-164.ap-northeast-2.compute.internal>
---
 config/cross-android-arm64.cbc     | 3 ++-
 config/cross-android-armv7.cbc     | 3 ++-
 config/cross-android-universal.cbc | 3 ++-
 3 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/config/cross-android-arm64.cbc b/config/cross-android-arm64.cbc
index ab90259..b469f97 100644
--- a/config/cross-android-arm64.cbc
+++ b/config/cross-android-arm64.cbc
@@ -3,7 +3,8 @@ from cerbero.config import Platform, Architecture, Distro, DistroVersion
 
 target_platform = Platform.ANDROID
 target_distro = Distro.ANDROID
-target_distro_version = DistroVersion.ANDROID_LOLLIPOP
+target_distro_version = DistroVersion.ANDROID_NOUGAT
+# target_distro_version = DistroVersion.ANDROID_LOLLIPOP
 target_arch = Architecture.ARM64
 
 #variants = ['nodebug']
diff --git a/config/cross-android-armv7.cbc b/config/cross-android-armv7.cbc
index 5820edc..33334d8 100644
--- a/config/cross-android-armv7.cbc
+++ b/config/cross-android-armv7.cbc
@@ -3,7 +3,8 @@ from cerbero.config import Platform, Architecture, Distro, DistroVersion
 
 target_platform = Platform.ANDROID
 target_distro = Distro.ANDROID
-target_distro_version = DistroVersion.ANDROID_GINGERBREAD
+target_distro_version = DistroVersion.ANDROID_NOUGAT
+# target_distro_version = DistroVersion.ANDROID_GINGERBREAD
 target_arch = Architecture.ARMv7
 
 #variants = ['nodebug']
diff --git a/config/cross-android-universal.cbc b/config/cross-android-universal.cbc
index 16a78af..0aa98d3 100644
--- a/config/cross-android-universal.cbc
+++ b/config/cross-android-universal.cbc
@@ -3,7 +3,8 @@ from cerbero.config import Platform, Architecture, Distro, DistroVersion
 
 target_platform = Platform.ANDROID
 target_distro = Distro.ANDROID
-target_distro_version = DistroVersion.ANDROID_GINGERBREAD
+target_distro_version = DistroVersion.ANDROID_NOUGAT
+#target_distro_version = DistroVersion.ANDROID_GINGERBREAD
 target_arch = Architecture.UNIVERSAL
 
 universal_archs = {
-- 
2.7.4

